cheatsheets/DotNet_Security_Cheat_Sheet.md

```diff
@@ -274,25 +274,23 @@ process.Start();
 
 DO: Use whitelist validation on all user supplied input. Input validation prevents improperly formed data from entering an information system. For more information please see the [Input Validation Cheat Sheet](Input_Validation_Cheat_Sheet.md).
 
-e.g Validating user input using [IPAddress.Parse Method](https://docs.microsoft.com/en-us/dotnet/api/system.net.ipaddress.parse?view=netframework-4.8)
+e.g Validating user input using [IPAddress.TryParse Method](https://docs.microsoft.com/en-us/dotnet/api/system.net.ipaddress.tryparse?view=netframework-4.8)
 
 ``` csharp
 //User input
 string ipaddress = "127.0.0.1";
  
 //check to make sure an ip address was provided    
 if (string.IsNullOrEmpty(address))  
-{   
-	try
+{
+	// Create an instance of IPAddress for the specified address string (in 
+	// dotted-quad, or colon-hexadecimal notation).
+	if IPAddress.TryParse(ipAddress, out var address)
```

Comment:
here is also type `ipAddress ` should be `ipaddress`

Reply:
My bad, I didn't see it. I fixed it.

Actually, the typo was already there before.

Reply:
There are missing brackets